releases/notice/sha256:23df717980b4aa08d2da6c4cfa327f1b730d92ec9cf740959d2d5911830d82fb.json.txt

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
How will these `sha256:<sha>.json.txt` files be maintained over time? Can we throw them away and simply keep the `notice.txt` file?

Reply:
those files avoid the need to execute tern more than once for each image, producing them took a few hours, if this was a quicker process I do not see a need to keep them around
I can store them in a different place

Reply:
Cool, yeah, I think I prefer keeping them out of the codebase, but if it makes your pipeline more difficult to maintain we can revisit that